ansible_collections/arista/avd/molecule/eos_validate_state/intended/structured_configs/dc1-leaf1a.yml

```diff
@@ -379,6 +379,30 @@ ethernet_interfaces:
   channel_group:
     id: 5
     mode: active
+- name: Ethernet31
```

Comment:
so here is it correct understanding to assume there is no updates to catalog because everything in added in the PR is asking to not be validated?

Reply:
Correct. I added ports with `validate_state: false` to make sure they are not added to the catalog.

Notice there are no changes for other devices structured_config, `validate_state: true` is not shown (related to your other comment).